change/@microsoft-teams-js-3e90cf0a-ad19-48b6-aa04-d66b3595072e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added beta tag to barCode capability to accurately reflect level of support offered",
```

Comment:
Can you please add backticks to "barCode"?

Reply:
haha just pushed